lib/authlogic/session/base.rb

```diff
@@ -2089,10 +2089,14 @@ def update_login_ip_addresses
         end
       end
 
+      def a_r_default_timezone
+        (ActiveRecord.respond_to?(:default_timezone) ? ActiveRecord : klass).default_timezone
+      end
+
```

Comment:
Can you please add a comment explaining which versions of AR this method is handling? A link to the AR changelog would be ideal, but not necessary.

Reply:
Or rather, can we merge https://github.com/binarylogic/authlogic/pull/744 instead of this? The former seems simpler, and matches #743.

Reply:
It seems this method is no longer used, and can be deleted?